### PR TITLE
Resolve 416 "Cargo impossible bounties": Disable Mining Bounties

### PR DESCRIPTION
**Pull request by Tennessee116**
## About The Pull Request

These commits disable mining bounties, resolving #416. 
## Why It's Good For The Game

Cargo needs to be able to complete their bounties in order to get more bounties and therefore more station credits, affecting the outcome of the round.

## Changelog
:cl: Tennessee116
tweak: Disabled mining bounties
/:cl:

### CODE REVIEW

File: nsv13.dme [hunk 1]
```diff
@@ -1568,7 +1568,7 @@
 #include "code\modules\cargo\bounties\item.dm"
 #include "code\modules\cargo\bounties\mech.dm"
 #include "code\modules\cargo\bounties\medical.dm"
-#include "code\modules\cargo\bounties\mining.dm"
+//#include "code\modules\cargo\bounties\mining.dm" //NSV13
```
Comment (francinum) [inline nsv13.dme]: This doesn't work. The compiler will automatically clear this line. Simply leave the file unticked.

File: code/modules/cargo/bounties/mining.dm [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (francinum) [inline code/modules/cargo/bounties/mining.dm]: There's no need to comment out the entire file.

File: nsv13.dme (hunk 1 above)
Comment (Tennessee116) [reply nsv13.dme]: so just leave #include "code\modules\cargo\bounties\mining.dm" be?

File: code/modules/cargo/bounties/mining.dm (hunk 2 above)
Comment (Tennessee116) [reply code/modules/cargo/bounties/mining.dm]: Alright. Would I just disable it from generating as a bounty in `bounty.dm` and leave everything else be?

**Pull request merged**